StitchCore/StitchCoreTests/StitchCoreTests.swift

```diff
@@ -86,7 +86,7 @@ class StitchCoreTests: XCTestCase {
             expectation.fulfill()
         }
 
-        waitForExpectations(timeout: 20, handler: nil)
+        waitForExpectations(timeout: 200, handler: nil)
```

Comment:
Is there a better way to not wait?

Reply:
As long as our tests are running asynchronous code, `waitForExpectations` is the way to do this with `XCTest`

Reply:
If we want, I could write another `await` block, but I'd like to keep this as close to the standards as possible while the language is still young.

Reply:
of course. just curious